ci-operator/step-registry/optional-operators/cvp-common/scorecard/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+  - dirgim
+  - petr-muller
```

Comment:
I don't want nor need to be here.

Can this have 2 approvers from CVP and possibly more CVP people in `revievers`, too?

Reply:
Alright, I will add some of my team members here if thats ok.

Reply:
@petr-muller we'd still like to have members of DPTP in the list of `reviewers` at least until the rehearsal jobs are set up.